pimd/pim_cmd_common.c

```diff
@@ -566,6 +566,13 @@ int pim_process_rp_cmd(struct vty *vty, const char *rp_str,
 		return CMD_WARNING_CONFIG_FAILED;
 	}
 
+#if PIM_IPV == 6
+	if (IN6_IS_ADDR_LINKLOCAL(&rp_addr)) {
```

Comment:
Can it be only global unicast or multicast as well?

Reply:
It should be global unicast address always.

Reply:
We can configure RP as loopback.
I have added the check to avoid multicast and unspecified address as RP.